services/fxa/subhandler.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import os
+import sys
+
+# import newrelic.agent
+import serverless_wsgi
+
+serverless_wsgi.TEXT_MIME_TYPES.append("application/custom+json")
+
+from os.path import join, dirname, realpath
+# First some funky path manipulation so that we can work properly in
+# the AWS environment
+sys.path.insert(0, join(dirname(realpath(__file__)), 'src'))
+
+# newrelic.agent.initialize()
```

Comment:
Do we want to keep this for followup or nuke it?

Reply:
needs to be wired